.gitignore

```diff
@@ -28,6 +28,7 @@ bld/
 .vs/
 # Uncomment if you have tasks that create the project's static files in wwwroot
 #wwwroot/
+Migrations/
```

Comment:
Ah, Ok. That won't work once there's a live DB though (i.e. migrations will need to be sorted), but it'll do for dev-only testing at the moment.